src/main/java/com/webapp/TextBook/validation/ApiValidation/StudnetBookInfoValidation/StudentBookInfoValidationImpl.java

```diff
@@ -0,0 +1,234 @@
+package com.webapp.TextBook.validation.ApiValidation.StudnetBookInfoValidation;
+
+import com.webapp.TextBook.validation.ApiValidation.ApiValidationHandler.ApiValidationHandler;
+import com.webapp.TextBook.validation.ApiValidation.StudentInfoValidation.StudentInfoValidationImpl;
+import com.webapp.TextBook.validation.Shared.ErrorBinding;
+import com.webapp.TextBook.validation.Shared.ErrorBindingException;
+import com.webapp.TextBook.validation.Shared.RegexPatternContainer;
+import com.webapp.TextBook.validation.Shared.SharedValidationState;
+import com.webapp.TextBook.viewModel.apiViewModel.StudentBookInfo;
+import com.webapp.TextBook.viewModel.apiViewModel.StudentInfo;
+import org.javatuples.Triplet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.configurationprocessor.json.JSONArray;
+import org.springframework.boot.configurationprocessor.json.JSONException;
+import org.springframework.boot.configurationprocessor.json.JSONObject;
+import org.springframework.stereotype.Component;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.Validator;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.Optional;
+
+/***
+ * This class is responsible for validaiing student book info
+ */
+@Component
```

Comment:
component?
Is this class injected somewhere? If not then we should remove this

Reply:
https://github.com/NicholasPierce1/TextBookServices/commit/a9066b134487241f2cc7f31f24a6e9d08d0a16aa